packages/edit-site/src/components/sidebar-navigation-screen-templates/style.scss

```diff
@@ -1,4 +1,11 @@
-.edit-site-sidebar-navigation-screen-templates__see-all {
-	/* Overrides the margin that comes from the Item component */
-	margin-top: $grid-unit-20 !important;
+.edit-site-sidebar-navigation-screen-templates__templates-list {
+	div:last-child {
+		position: sticky;
+		bottom: 0;
+		margin-bottom: - $grid-unit-20;
+		padding: $grid-unit-20 0;
+		background-color: $gray-900;
+		margin-top: $grid-unit-20;
+		box-shadow: 0 -#{$grid-unit-10} $grid-unit-20 $gray-900;
```

Comment:
it would be nice for this overflow styling to be on the list instead and pulled out into a more generic class so we can apply it to any list that overflows

Reply:
Actually, probably should be using [Scrollable](https://wordpress.github.io/gutenberg/?path=/story/components-experimental-scrollable--default) here and extending to add prop for showing shadow or not.